blogpost.txt

```diff
@@ -0,0 +1,40 @@
+What is a ruby gem? 
```

Comment:
As a suggestion, it would be great practice to try using Markdown for articles like this. You will use it again and again when creating README.md files and it's fairly essential for a dev to know, so may as well get as much practice in as possible :grin: